packages/mattermost-redux/src/utils/channel_utils.ts

```diff
@@ -42,6 +42,10 @@ export function completeDirectChannelInfo(usersState: UsersState, teammateNameDi
     return channel;
 }
 
+export function splitRoles(roles: string): Set<string> {
```

Comment:
0/5, but this could live next to the roles reducer I don't think we need to preemptively export helpers like this for use in other code since it leads to more inter-dependency between files.